### PR TITLE
Build ES Accordion Component

### PR DESCRIPTION
**Pull request by pbishop16**
Addresses #17 

- Keyboard Interactions implemented: `Enter`, `Space`, `Up Arrow`, `Down Arrow`, `Home` and `End`

- @MelSumner I added a temporary example in the `index.hbs` so you can validate that it fits and/or provide guidance on corrections. Will remove the example once approved.

- Footer still needed.

image

### CODE REVIEW

**Comment by MelSumner** [comment]
hey @locks you'll like this one I think

File: addon/styles/_es-accordion.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+				border-color: transparent;
+				color: $gray;
+				height: 36px;
+				width: 36px;
+				&:hover {
+					cursor: pointer;
+				}
+			}
+		}
+		.accordion-body {
```
Comment (MelSumner) [inline addon/styles/_es-accordion.scss]: need to be a tiny bit picky here, can you please alpha-sort the CSS properties? thank you!

File: addon/templates/components/es-accordion.hbs [hunk 2]
```diff
@@ -0,0 +1 @@
+{{yield accordionState}}
```
Comment (MelSumner) [inline addon/templates/components/es-accordion.hbs]: do you think we should have it yield a block so anything can be put inside?

File: addon/styles/_es-accordion.scss (hunk 1 above)
Comment (pbishop16) [reply addon/styles/_es-accordion.scss]: Nothing too picky. Absolutely will do.

Comment (pbishop16) [reply addon/styles/_es-accordion.scss]: Completed.

File: addon/templates/components/es-accordion.hbs (hunk 2 above)
Comment (pbishop16) [reply addon/templates/components/es-accordion.hbs]: Current approach has been discussed and accepted.

**Comment by MelSumner** [comment]
@pbishop16 you can merge this if you're ready! Thank you for the contribution!

**Comment by pbishop16** [comment]
@MelSumner Greatly appreciated. Will do.

**Pull request merged**